### PR TITLE
:recycle:  Move text area field to a text message component for reuse 

### LINKED ISSUE #316: ♻️ 🎨 Move the message (text area) field of blocks to a separate component for reuse
## User stories

This is a refactor, and requires the messages of story blocks to be moved to a separate component. This will ensure reusability and easy maintainability.

---

Quick links: [Summary](#summary) | [Description](#description) | [Template](#template) | [Example](#example) | [Resources](#resources)

---

### Description

This separate component for now will only have the `textarea` html element as implemented in the blocks that have a message. 

Example:
The `MessageBlockComponent` has this for its message field: `<textarea id="message" name="message" formControlName="message" type="text" rows="3"></textarea>`. This should be moved to the `TextMessageComponent` and replace the text area tag with the new component tag.

Therefore you will need to create a `TextMessageComponent` on this path: `libs/features/convs-mgr/stories/blocks/library/main/src/lib/components/text-message-block`. 

Once that is done, replace all message fields with the new component tag, and take note of the inputs passed.

### Acceptance Criteria

1. There should be one reusabe component for text message fields.
2. The input bindings should applied down to the child `textarea` tag as before
3. The application and the styling should work exactly as before.

### PR DESCRIPTION
**Pull request by aboiyon**
# Description

This is a refactor where the messages of story blocks are moved to a separate component TextMessageComponent. This will ensure reusability across the project where we have 'textarea'.

Fixes #316 

# Screenshot

### CODE REVIEW

File: libs/features/convs-mgr/stories/blocks/library/list-message-block/src/lib/components/list-block/list-block.component.html [hunk 1]
```diff
@@ -1,7 +1,9 @@
 <div [id]="id" *ngIf="listMessageBlock">
   <form [formGroup]="listMessageBlock" fxLayout="column" fxLayoutALign="start center">
 
-    <textarea id="message" name="message" formControlName="message" type="text" rows="3"></textarea>
+    <!-- <textarea id="message" name="message" formControlName="message" type="text" rows="3"></textarea> -->
+
```
Comment (ReaganCn) [inline libs/features/convs-mgr/stories/blocks/library/list-message-block/src/lib/components/list-block/list-block.component.html:5]: Remove unused commented code

File: libs/features/convs-mgr/stories/blocks/library/multiple-input-message-block/src/lib/components/multiple-input-block/multiple-input-block.component.html [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
-    <textarea id="addMore" name="addMore" formControlName="addMore" type="text" rows="1"
-      placeholder="Add more message"></textarea>
+    <!-- <textarea id="message" name="message" formControlName="message" type="text" rows="1"
+      placeholder="Message"></textarea> -->
+
+    <app-text-message></app-text-message>
+
+    <!-- <textarea id="addMore" name="addMore" formControlName="addMore" type="text" rows="1"
+      placeholder="Add more message"></textarea> -->
+
```
Comment (ReaganCn) [inline libs/features/convs-mgr/stories/blocks/library/multiple-input-message-block/src/lib/components/multiple-input-block/multiple-input-block.component.html:11]: Same here. Do the same for all the commented out code in this PR

**Review by ReaganCn — CHANGES_REQUESTED** [review]
Good job, just some minor cleanup and you're good

**Review by ReaganCn — CHANGES_REQUESTED** [review]
I cannot see the code for the TextMessageComponent

**Comment by aboiyon** [comment]
> I cannot see the code for the TextMessageComponent

https://github.com/italanta/elewa/pull/318/commits/da589c48e68a6282596d7231eb87730c5e80bea1 here is the code

**Review by ReaganCn — CHANGES_REQUESTED** [review]
Please make sure that the text message component is in the path specified in the issue.
It should be in: 
image

**Review by ReaganCn — APPROVED** [review]

**Pull request closed without merging**